qdrant-landing/content/documentation/quick-start.md

```diff
@@ -7,8 +7,9 @@ aliases:
 # Quickstart
 
 In this short example, you will use the Python Client to create a Collection, load data into it and run a basic search query. 
+The first few steps will show you how to self-host Qdrant using Docker. However, you can always sign up for [Qdrant Cloud](../cloud/quickstart-cloud/) and spin up a cluster with much less effort. Your first cluster is completely free.
```

Comment:
I would only keep this line

Reply:
I suggest pushing an absolute link, in case we move this page in the future.

```suggestion
The first few steps will show you how to self-host Qdrant using Docker. However, you can always sign up for [Qdrant Cloud](/documentation/cloud/quickstart-cloud/) and spin up a cluster with much less effort. Your first cluster is completely free.
```